packages/expo/ios/AppDelegates/EXAppDelegateWrapper.mm

```diff
@@ -70,9 +70,6 @@ - (void)applicationDidBecomeActive:(UIApplication *)application
 #else
 - (void)applicationDidFinishLaunching:(NSNotification *)notification
 {
-#if __has_include(<ReactAppDependencyProvider/RCTAppDependencyProvider.h>)
```

Comment:
this is unrelated to the fix - this code is for mac which wasn't tested due to not being compatible with 78. Anyways, it's not supposed to be here

Reply:
cc @gabrieldonadel